compute/cker/include/cker/operation/Conv.h

```diff
@@ -83,27 +83,32 @@ class Conv
 
   void operator()(const ConvParams &params, const Shape &input_shape, const float *input_data,
                   const Shape &filter_shape, const float *filter_data, const Shape &bias_shape,
-                  const float *bias_data, const Shape &output_shape, float *output_data)
+                  const float *bias_data, const Shape &output_shape, float *output_data,
+                  ruy::Context *ruy_context)
   {
-    if (usableMultiThreaded(params.padding_type, params.dilation_width_factor,
-                            params.dilation_height_factor))
+    if (!_prepared)
     {
-      bool transposed_in_execution = false;
-      if (!_prepared)
-      {
-        // This means that filter is not constant
-        // TODO Apply optimized kernel if multithreaded kernel is slower than optimized kernel by
-        // transposing filter data
-        transposeFilter(filter_shape, filter_data, transposed_in_execution);
-      }
-      multithreaded::Conv(params, input_shape, input_data, filter_shape, &_modified_filter_data[0],
-                          bias_shape, bias_data, output_shape, output_data);
+      // This means that input or output are dynamic or filter is not constant
+      IsRequiredIm2col(input_shape, filter_shape, output_shape, params.stride_width,
```

Comment:
Conditions to use `Im2col` has been changed, so this function should be updated.